cmd/standalone/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM golang:1.7
+EXPOSE 5775 6831 6832 14267 3000 3001 8080 8081 8082 8083
+
+ADD . /go/src/github.com/uber/jaeger
```

Comment:
I didn't like what I for crossdock, I first wanted to build everything inside the container but couldn't figure out how. I prefer building everything inside the container instead of doing it in travis and passing the binaries. I see no benefit of having travis do it.